03 Writing Algorithms/12 Universes/03 Equity/01 Liquidity Universes/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "metadata",
+    "values": {
+        "description": "You can select a set of stocks with the greatest dollar volume in the market.",
```

Comment:
I think this description should be a definition. "A liquidity universe lets you select a set of stocks with the greatest dollar volume in the market."